### ISSUE TITLE
Soundflower does not work with mac os 10.6.2 and ishowu v1.75

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mattscotthudson/soundflower#51]
```
What steps will reproduce the problem?
1. Install soundflower 1.5.1
2. Use with ishowu 1.75
3. and using snow leopard 10.6.2

What is the expected output? What do you see instead?
Cannot get the system audio to get into ishow.  Records video with no sound.  
It seems that even 
though soundflower is shown in the sound preferences it does not seem to want 
to kickon and if 
I toggle between the system sound panel off and on it always goes back to the 
external speakers 
output vs. staying on 2-channel soundflower.


What version of the product are you using? On what operating system?
Soundflower 1.5.1, mac ox 10.6.2, and ishowu 1.75


Please provide any additional information below.


```

Original issue reported on code.google.com by `williamd...@gmail.com` on 16 Nov 2009 at 4:30

**Comment by GoogleCodeExporter** [comment]
```
I also cannot get soundflower 1.5.1 to work after the update to OS 10.6.2. 

Steps to reproduce the problem:

1. Audio Hijack 2.6.9 fails to record using *any* method with Soundflower 
installed. 
2. Uninstall Soundflower
3. Audio Hijack will now record using an external device, but will not record 
system
audio, for which it requires soundflower.

```

Original comment by `shambhav...@gmail.com` on 22 Nov 2009 at 6:36

**Comment by GoogleCodeExporter** [comment]
```
I am running:
- OS X 10.6.1
and
- ishowu 1.75

just upgraded to soundflower 1.5.1 and it works fine for me.
```

Original comment by `chr...@gmail.com` on 23 Nov 2009 at 7:41

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
i am having the same exact problem! please try to fix this as fast as possible!
```

Original comment by `GrantGra...@gmail.com` on 8 Jan 2010 at 1:05

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
hmm.. I had soundflower and couldn't record any audio - internal mic or usb 
lead on guitar
uninstalled it via the applescript and it still doesn't work.
Any ideas anyone?
```

Original comment by `d...@vidbeamish.co.uk` on 17 Feb 2010 at 10:02

**Comment by GoogleCodeExporter** [comment]
```
bump.

I wanna use a custom voicemail greeting with skype, and the fail of this 
prevents me
from doing that.  That makes me frowny.  Plzfixkthxbye.
```

Original comment by `mathew.h...@gmail.com` on 20 Feb 2010 at 5:53

**Comment by GoogleCodeExporter** [comment]
```
same probs

```

Original comment by `vonwe...@gmail.com` on 2 Apr 2010 at 10:44

**Comment by GoogleCodeExporter** [comment]
```
Don't know if this will help, but you can try it. The issue in another report 
seems related, may not work, but it's worth a try. Look for the post's I just 
made (compare time stamps with this comment). 

http://code.google.com/p/soundflower/issues/detail?id=62
```

Original comment by `chrisric...@gmail.com` on 22 Jun 2010 at 6:38

**Comment by GoogleCodeExporter** [comment]
```
Im running 10.6.4 snow leopard and I cant get soundflower to work with any apps 
including ishowu.  I think it may be a system prefs issue or something because 
when i change the output and input it doesnt stick.  The sound is not rerouted 
and once i close system prefs then reopen the out/inputs just default back to 
internal speakers etc.    
```

Original comment by `aussiesu...@yahoo.com.au` on 24 Jun 2010 at 6:47

**Comment by GoogleCodeExporter** [comment]
```
I get this error message while installing 1.5.1, maybe if someone who has 
Soundflower working hosted this file (assuming its the same error for everyone) 
then we could manually replace the corrupt file and get it working? 
```

Original comment by `Loubat...@gmail.com` on 31 Jul 2010 at 9:02

Attachments:
- [Screen shot 2010-07-31 at 10.00.20.png](https://storage.googleapis.com/google-code-attachments/soundflower/issue-51/comment-11/Screen shot 2010-07-31 at 10.00.20.png)

**Comment by GoogleCodeExporter** [comment]
```
I am running 10.6.6 and can't get it to work either. 
```

Original comment by `serendip...@gmail.com` on 16 Mar 2011 at 8:24

**Comment by GoogleCodeExporter** [comment]
```
SAme error message, same problem. I can't use it :(
```

Original comment by `natha1...@gmail.com` on 10 Apr 2011 at 5:47

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
had same problem but 10.5.2 is now available and works fine.
```

Original comment by `anaba...@gmail.com` on 4 Jun 2011 at 3:01

**Comment by GoogleCodeExporter** [comment]
```
I am using Snow Leopard 10.6.8 Was having difficulty configuring 
Soundflowerbed. Out of frustration I have uninstalled it and reinstalled it 
(including downloading it again). Now I get the Sunflower icon but when I click 
on it, it tells me Soundflower is not installed.

```

Original comment by `f134kil...@comcast.net` on 1 Jul 2011 at 7:50

**Comment by GoogleCodeExporter** [comment]
```
Same issue as comment 16.
```

Original comment by `volker.h...@gmail.com` on 29 Jul 2011 at 10:40

**Comment by GoogleCodeExporter** [comment]
```
I got the Soundflower not installed msg at first too. Restart the machine and 
that msg will go away and you sill see all the Soundflower functions but then 
it still won't work under 10.6.8. I tried recording with Soundtrack Pro and 
Adobe Soundbooth and I get no audio. I switched to the built-in mic and it 
recorded audio but when I switch the input back to Soundflower(2ch) I get no 
audio.
```

Original comment by `j...@Austin.rr.com` on 1 Nov 2011 at 5:22

**Comment by GoogleCodeExporter** [comment]
```
I am Also have problem with Soundflower 1.5.2 on Macbook pro 10.6.8, I am has 
select soundflower as output on system preference sound panel and as sound 
input on max msp 6, but I dont hear sounds coming out. What wrong with 
soundflower?
```

Original comment by `niejo...@gmail.com` on 30 Jan 2012 at 6:37

**Comment by GoogleCodeExporter** [comment]
```
There are a variety of different reports here which don't all seem related.  At 
least some of them should be addressed at 
https://github.com/tap/Soundflower/wiki/Frequent-Questions .

Some others may be addressed in the new version 1.6 release candidate.



```

Original comment by `t...@electrotap.com` on 3 Feb 2012 at 9:33
- Changed state: **Done**
